src/server.js

```diff
@@ -100,6 +113,14 @@ in ${config.public ? "public" : "private"} mode`);
 
 		manager.autoloadUsers();
 	}
+
+	function shutdown() {
+		log.info("Quitting...");
+		server.close(); // Removes socket file
+		process.exit();
+	}
+
+	process.on("SIGINT", shutdown);
```

Comment:
I understand why it is necessary. I'm just saying it disable node's default behaviour.

http://stackoverflow.com/a/14372355/2200891
http://stackoverflow.com/a/14032965/2200891